index.html

```diff
@@ -207,10 +212,10 @@ <h1 class="skills-header"><span><img src="./images/works/Rectangle-three.svg" al
         <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
     </div>
     <div class="form-container">
-        <form action="POST">
-            <input type="text" placeholder="Full name">
-            <input type="email" placeholder="Email">
-            <textarea name="contact" id="text-msg" cols="30" rows="5" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/mknkennj">
+            <input type="text" name="user_name" id="name" maxlength="30" placeholder="Enter your name" required>
+            <input type="email" name="Email" id="email" placeholder="Enter your email" required>
+            <textarea name="contact" id="text-msg" placeholder="Enter text here" maxlength="500" required></textarea>
             <input type="button" value="Get in touch" id="btn-submit">
```

Comment:
Kindly work on these changes:
- Can't submit the form. Fix this issue.
![Screen Shot 2021-09-28 at 17 59 54](https://user-images.githubusercontent.com/22258274/135176928-296788d0-782b-41ea-9581-a610abb250ee.png)